docs(config): add Elasticsearch instructions to Connecting to the Database (#2888)
cube-js/cube.js: docs/content/Configuration/Connecting-to-the-Database.md

CUBEJS_DB_DATABRICKS_URL=jdbc:spark://dbc-XXXXXXX-XXXX.cloud.databricks.com:443/default;transportMode=http;ssl=1;httpPath=sql/protocolv1/o/XXXXX/XXXXX;AuthMech=3;UID=token;PWD=XXXXX
```

### Elasticsearch

> To connect to a Elasticsearch database use
> `CUBEJS_DB_URL` with the username and password embedded in the url if required
> `CUBEJS_DB_ELASTIC_QUERY_FORMAT` is required if it is not a elastic cloud client

Reviewer: ```suggestion

To connect to a Elasticsearch database, use `CUBEJS_DB_URL` with the username and password embedded in the URL, if required. If you're not using Elastic Cloud, you **must** specify `CUBEJS_DB_ELASTIC_QUERY_FORMAT`.
```
